packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -154,9 +171,9 @@ export default class SdNavigationItem extends SolidElement {
         target=${ifDefined(isLink ? this.target : undefined)}
         download=${ifDefined(isLink ? this.download : undefined)}
         rel=${ifDefined(isLink && this.target ? 'noreferrer noopener' : undefined)}
-        role=${isLink ? 'link' : 'button'}
-        tabindex=${this.disabled ? '-1' : '0'}
-        @click=${isAccordion ? this.handleClickSummary : isButton ? this.handleClickButton : undefined}
+        role=${!this.separated ? (isLink ? 'link' : 'button') : 'null'}
+        tabindex=${this.disabled || this.separated ? '-1' : '0'}
+        @click=${!this.separated ? (isAccordion ? this.handleClickSummary : isButton ? this.handleClickButton : undefined) : undefined}
```

Comment:
Personal taste: an exclamation mark (!) is easily overlooked. I would invert the if statement to prevent future reading errors.

Reply:
Interesting perspective! :)